packages/ndla-ui/src/Notion/ConceptNotion.tsx

```diff
@@ -4,79 +4,182 @@
  * This source code is licensed under the GPLv3 license found in the
  * LICENSE file in the root directory of this source tree. *
  */
+import styled from '@emotion/styled';
 
 import React, { useEffect } from 'react';
 //@ts-ignore
 import { initArticleScripts } from '@ndla/article-scripts';
-import { Notion } from '.';
+import { useTranslation } from 'react-i18next';
+import { breakpoints, mq, spacing } from '@ndla/core';
+import Notion, { NotionDialogContent, NotionDialogText, NotionDialogLicenses } from '@ndla/notion';
+import { Notion as UINotion } from '.';
 import { NotionImage } from './NotionImage';
 import NotionVisualElement, { NotionVisualElementType } from './NotionVisualElement';
 import FigureNotion from './FigureNotion';
 import { Copyright } from '../types';
 
+const ContentWrapper = styled.div<{ adjustSizeToFitWiderPage?: boolean }>`
+  position: relative;
+  right: auto;
+  left: ${(props) => (props.adjustSizeToFitWiderPage ? '0%' : '-16.6666666667%')};
+  width: ${(props) => (props.adjustSizeToFitWiderPage ? '100%' : '133.3333333333%')};
+  padding-left: 24px;
+  padding-right: 24px;
+  & button {
+    width: 100%;
+  }
```

Comment:
Dette er stylingen som gjør at du må bruke !important for å fikse ekspander-knappen. Om dette fjernes eller gjøres mer presist vil du slippe bruk av important